Dockerfile

```diff
@@ -63,6 +68,15 @@ RUN yum install -y \
     python-dateutil \
     && yum clean all
 
+#Hide default http welcome page
+RUN mv /etc/httpd/conf.d/welcome.conf /etc/httpd/conf.d/welcome.bak
+
+#Modify the config to allow us to restric access to root html directory
+RUN sed -i '151s/.*/    AllowOverride All/' /etc/httpd/conf/httpd.conf
```

Comment:
This could be brittle to changes in /etc/httpd/conf/httpd.conf e.g. a new line added above this one. I'm not a sed guru though.

Reply:
I have amended the change to first look for all the `AllowOverride All` strings in the file and then change the 2nd one it finds. Although this can still break if another similar section is added to the file it will be more resilient to more general changes that cause the line numbering to change.